src/main/java/com/synopsys/integration/alert/web/config/JobConfigActions.java

```diff
@@ -129,8 +131,13 @@ public void deleteJobById(UUID id) throws AlertException {
     }
 
     public JobFieldModel saveJob(JobFieldModel jobFieldModel) throws AlertException {
-        validateJob(jobFieldModel);
+        MessageResult validationResult = validateJob(jobFieldModel);
+        List<AlertFieldStatus> fieldStatuses = validationResult.getFieldStatuses();
+        if (!fieldStatuses.isEmpty()) {
+            throw new AlertFieldException(fieldStatuses);
+        }
```

Comment:
This is done in multiple places, we should add a method that performs this check and throws the exception to the MessageResult object.

Reply:
I was hesitant to do this because I eventually want to get rid of AlertFieldException and just return the results object.

Reply:
I agree with that sentiment, but it doesn't hurt to have the method right now then remove it when it's no longer needed